<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress Bar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 255);color:#ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{background-color:rgb(0, 65, 197);text-align:center;color:#000000;border-radius:10px;font-weight:bold;}QProgressBar::chunk{background-color:rgb(85, 255, 0);border-radius:10px;}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#FFFFFF;color:#000000;border-radius:20px;font-weight:bold;}QPushButton:hover{background-color:#c4c4c4;}QPushButton:pressed{background-color:#999999;}</string>
    </property>
    <property name="text">
     <string>Aumentar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>230</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#FFFFFF;color:#000000;border-radius:20px;font-weight:bold;}QPushButton:hover{background-color:#c4c4c4;}QPushButton:pressed{background-color:#999999;}</string>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
